### PR TITLE
Fix for missing imports in modals.less

### PR DESCRIPTION
**Pull request by TotomInc**
## Brief description

`modals.less` have missing imports, which breaks `gulp build`. This is been fixed and need to prepare papercss for 1.3.1, as said in #116. This needs to be merged ASAP.

Waiting for @rhyneav to prepare for 1.3.1. Once you have added your commits, this can be merged without any errors/problems.

## Developer Certificate of Origin

- [x] I certify that these changes according to the Developer Certificate of Origin 1.1 as described at <https://developercertificate.org/>.

### CODE REVIEW

**Comment by afzalsayed96** [comment]
Thanks for the fix! Somehow I missed it.

File: src/modals.less [hunk 1]
```diff
@@ -15,24 +19,24 @@
 }
```
Comment (afzalsayed96) [inline src/modals.less]: Hey, I realized that this piece of code is redundant and not being used anywhere which I totally forgot to remove before pushing it. Can you please remove this?
```
 .transition(@transition) {
   -webkit-transition: @transition;
   transition: @transition;
}
```

Comment (TotomInc) [reply src/modals.less]: I'll do it ASAP

Comment (afzalsayed96) [reply src/modals.less]: Thanks 👍

Comment (TotomInc) [reply src/modals.less]: And it's done!

**Comment by TotomInc** [comment]
@rhyneav can you please merge this PR? It ready for 1.3.1, I've also bumped the `package.json` to 1.3.1 so it should be ready.

File: src/modals.less (hunk 1 above)
Comment (rhyneav) [reply src/modals.less]: Good catch!

**Comment by rhyneav** [comment]
You rock! Thank you for the quick catch and the update. Everything is updated and released on Github and NPM for version 1.3.1

**Pull request merged**